mistral_example.py

```diff
@@ -47,6 +50,9 @@
     answer = answer.replace("\n", "\\n")
     print( answer )
     print( "-----------------------------------\n" )
+    
+    if torch.backends.mps.is_available():
+        torch.mps.empty_cache()
```

Comment:
Seems like a pytorch issue in https://github.com/pytorch/pytorch/issues/105839 ?